PLUB/Sources/Views/Home/MainPage/Cell/TodoCollectionViewCell.swift

```diff
@@ -74,29 +89,32 @@ final class TodoCollectionViewCell: UICollectionViewCell {
     [profileImageView, likeButton, likeCountLabel, listContainerView, moreButton].forEach { contentView.addSubview($0) }
     
     profileImageView.snp.makeConstraints {
-      $0.top.leading.equalToSuperview()
+      $0.top.leading.equalToSuperview().inset(8)
       $0.size.equalTo(24)
     }
     
     likeButton.snp.makeConstraints {
-      $0.top.equalTo(profileImageView.snp.bottom).offset(10.67)
-      $0.trailing.equalTo(profileImageView.snp.centerX).offset(-1)
+      $0.top.equalTo(profileImageView.snp.bottom).offset(8)
+      $0.leading.equalToSuperview().inset(3)
+      $0.bottom.lessThanOrEqualToSuperview()
       $0.size.equalTo(16)
     }
     
     likeCountLabel.snp.makeConstraints {
-      $0.top.equalTo(likeButton)
+      $0.centerY.equalTo(likeButton)
       $0.leading.equalTo(likeButton.snp.trailing).offset(2)
     }
     
     listContainerView.snp.makeConstraints {
       $0.leading.equalTo(profileImageView.snp.trailing).offset(12)
-      $0.directionalVerticalEdges.trailing.equalToSuperview()
+      $0.bottom.lessThanOrEqualToSuperview()
+      $0.top.trailing.equalToSuperview()
     }
     
     moreButton.snp.makeConstraints {
       $0.size.equalTo(32)
-      $0.top.trailing.equalToSuperview().inset(8)
+      $0.top.equalToSuperview().inset(8)
+      $0.trailing.equalToSuperview().inset(7)
```

Comment:
이거 8로 잡으시면 될 거 같아요. 디자인 오류가 있는 듯 합니다.

```suggestion
      $0.top.trailing.equalToSuperview().inset(8)
```

Reply:
그니까요 저런 부분이 되게 많더라구요 수정하도록 하겠습니다 !!